style_editor.html

```diff
@@ -9312,6 +9314,23 @@
   button.style.backgroundColor = framesPerUpdate == 60 ? "dodgerblue" : "";
 }
 
+</script>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/FileSaver.js/2.0.2/FileSaver.min.js"></script>
+<!-- 
+MIT license available here: https://spdx.org/licenses/MIT.html?utm_source=cdnjs&utm_medium=cdnjs_link&utm_campaign=cdnjs_library
+https://github.com/eligrey/FileSaver.js/ -->
+<script>
+
+/* Save blade style to local storage via user OS dialog */
+function ClickSave() {
+  Copy();
+  let textArea = FIND("style");
+  let content = "Saved from ProffieOS Style Editor:\nhttps://fredrik.hubbe.net/lightsaber/style_editor.html" + "\n\n" + textArea.value;
```

Comment:
Seems like the header part should be inside a comment so it can be pasted directly into a config file.

Reply:
good idea. Edited.